packages/react/src/AdvancedVideo.tsx

```diff
@@ -115,7 +116,12 @@ class AdvancedVideo extends Component <VideoProps> {
       if (key in this.props) {
         result[key] = this.props[key];
       }
-    });
+    })
+    if (this.props.cldPoster === 'auto') {
+      result['poster'] = this.props.cldVid.quality("auto").format("jpg").toURL();
```

Comment:
We might also consider using so_auto:
```a constant value of auto (relevant only for video thumbnails). This value automatically selects a frame that best matches the average value of a color distribution analysis of the first few seconds of the video, making it more likely to display a typical frame.```
https://cloudinary.com/documentation/transformation_reference#so_start_offset

Reply:
@const-cloudinary this is strange, I don't see that text in the documentation page linked here.